docs/UserGuide.md

```diff
@@ -259,182 +277,191 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
+* If there are still contacts assigned, you can use `f/` to force the deletion of the `Wedding`. This will unassign all contacts from that `Wedding`.
 
 ## Task Features
 
-### Creating a Task : `create-task`
+### Creating a Task : `create-task` or `ctask`
 
 Creates a `Task` in WedLinker
 
-Format: `create-task tk/TASKDESCRIPTION [REMARKS]`
+Format: `create-task tk/TASK_DESCRIPTION [REMARKS]` or `ctask tk/TASK_DESCRIPTION [REMARKS]`
 
 * Tasks may have no dates, a single date indicating a deadline, or two dates to define a start and end period.
 * The dates can be specified under `REMARKS` with the format `d/YYYY-MM-DD`
 * Tasks are unique in WedLinker, there would not be any duplicated tasks.
-* Tasks can be assigned to a contact using the [assign-task](#assigning-a-task-to-a-contact--assign-task-) command.
+* `TASK_NAME` is case-insensitive.
+* Tasks can be assigned to a contact using the [assign-task](#assigning-a-task-to-a-contact--assign-task) command.
 
-### Delete a Task : `delete-task`
+### Deleting a Task : `delete-task` or `dtask`
 
 Deletes a `Task` from WedLinker
 
-Format: `delete-task INDEX`
+Format: `delete-task TASK_NUMBER` or `dtask TASK_NUMBER`
 
-* Deletes the specific `Task` at the INDEX when in [list-tasks](#listing-all-tasks--list-tasks) view.
+* Deletes the specific `Task` at the `TASK_NUMBER` when in [list-tasks](#listing-all-tasks--list-tasks) view.
 * The index **must be a positive integer** 1, 2, 3, …​.
 
-### Assigning a Task to a contact : `assign-task` 
+### Assigning a Task to a contact : `assign-task` or `atask`
 
-Format: `assign-task PERSONINDEX TASKINDEX`
+Format: `assign-task PERSON_NUMBER TASK_NUMBER` or `atask PERSON_NUMBER TASK_NUMBER`
 
 * Assigns a `Task` to a contact.
 * The indexes correspond to the indexes when in the [list-tasks](#listing-all-tasks--list-tasks) view.
-* The `PERSONINDEX` refers to the index of the person shown under the **People** column.
-* The `TASKINDEX` refers to the index of the task shown under the **Tasks** column
+* The `PERSON_NUMBER` refers to the index of the person shown under the **People** column.
+* The `TASK_NUMBER` refers to the index of the task shown under the **Tasks** column
 * The indexes **must be positive integers** 1, 2, 3, …​.
 
-### Un-assigning a Task from a contact : `unassign-task` 
+### Unassigning a Task from a contact : `unassign-task` 
 
-Format: `unassign-task PERSONINDEX TASKINDEX_OFPERSON`
+Format: `unassign-task PERSON_NUMBER TASK_NUMBER`
 
 * Un-assigns a `Task` from a contact.
-* The `PERSONINDEX` is the index of the person shown in the displayed person list.
-* The `TASKINDEX_OFPERSON` is the index of the task associated with the selected person.
+* The `PERSON_NUMBER` is the index of the person shown in the displayed person list.
+* The `TASK_NUMBER` is the index of the task associated with the selected person that will be unassigned.
```

Comment:
I think this point might be a little unclear, maybe you can say : "The `TASK_NUMBER` is the index of the task in the person's task list. For example, if Person 4 has 2 tasks assigned, you would use the following command to un-assign the person's second task: `unassign-task 4 2`."

Reply:
👍